### PR TITLE
SFP templates

### PR DESCRIPTION
**Pull request by ante185**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Adds AI faction templates featuring the SFP mod
2x olive/woodland templates, early and modern equipment respectively
2x snow templates, same as above, most vics in snow camo    

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Pending feedback and playtesting

********************************************************
Notes:
Now testable

### CODE REVIEW

**Comment by killerswin2** [comment]
So, we weren't gonna port this because the assets in it have problems, and are of low quality.

**Comment by killerswin2** [comment]
Also, this is second pr from you that notice extra commits at aren't your from you. You should check how you are branching.

**Comment by ante185** [comment]
I'll look into that

**Comment by Bob-Murphy** [comment]
> So, we weren't gonna port this because the assets in it have problems, and are of low quality.

What problems are there with the assets?

**Comment by ante185** [comment]
> > So, we weren't gonna port this because the assets in it have problems, and are of low quality.
> 
> What problems are there with the assets?

So i'd like to get some feedback then, so i know whether or not to put this on ice and tinker with other things instead

**Comment by ante185** [comment]
these templates should finished enough to have a surface level look over now.

With CUP the helicopter fleet got fleshed out and two extra APCs could be added.
Loadouts are all currently just a single step above the minimum needed to work - I'll be working on that next but as is these templates should all be functional now.

File: A3A/addons/core/Templates/Templates/SFP/SFP_AI_SWE.sqf [hunk 1]
```diff
@@ -0,0 +1,881 @@
...
+/////////////////////
+//Faces and Voices given to AI Factions.
+["faces", ["AfricanHead_01","AfricanHead_02","AfricanHead_03","Barklem",
+"GreekHead_A3_05","GreekHead_A3_06","GreekHead_A3_07","GreekHead_A3_08",
+"GreekHead_A3_09","Sturrock","WhiteHead_01","WhiteHead_02","WhiteHead_03",
+"WhiteHead_04","WhiteHead_05","WhiteHead_06","WhiteHead_07","WhiteHead_08",
+"WhiteHead_09","WhiteHead_10","WhiteHead_11","WhiteHead_12","WhiteHead_13",
+"WhiteHead_14","WhiteHead_15","WhiteHead_16","WhiteHead_17","WhiteHead_18",
+"WhiteHead_19","WhiteHead_20","WhiteHead_21"]] call _fnc_saveToTemplate;
+["voices", ["CUP_D_Male01_GB_BAF","CUP_D_Male02_GB_BAF","CUP_D_Male03_GB_BAF","CUP_D_Male04_GB_BAF","CUP_D_Male05_GB_BAF"]] call _fnc_saveToTemplate;
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/SFP/SFP_AI_SWE.sqf:91]: Swedish English accent tends to be quite far from the Bri'ish Accent

**Comment by Lazejun** [comment]
A funny side note with SFP Vehicles.

It takes only 1 Vehicle to unlock almost any Standard Issued Magazine for SFP Gear as they slap an excessive amount of magazines in each Vehicle.

One day we will strip Inventories and Slap a fixed amount of Faction specific Gear in there.

File: A3A/addons/core/Templates/Templates/SFP/SFP_AI_SWE.sqf (hunk 1 above)
Comment (ante185) [reply A3A/addons/core/Templates/Templates/SFP/SFP_AI_SWE.sqf:91]: True, but there doesn't appear to be any accents that would be less wrong either

Comment (Lazejun) [reply A3A/addons/core/Templates/Templates/SFP/SFP_AI_SWE.sqf:91]: Normal English?

Comment (ante185) [reply A3A/addons/core/Templates/Templates/SFP/SFP_AI_SWE.sqf:91]: American english? eh i guess so

Comment (ante185) [reply A3A/addons/core/Templates/Templates/SFP/SFP_AI_SWE.sqf]: American english applied

**Comment by ante185** [comment]
As done as can be for now, pending additional feedback.

**Comment by ante185** [comment]
Logistics nodes for vehicles added

**Comment by ante185** [comment]
With the airplane loadouts set i think that would be everything for these templates.
Should be good for review now.

**Comment by ante185** [comment]
Made a couple final changes and adjustments, merged latest unstable in, opened for review

**Comment by ante185** [comment]
Updated to latest unstable.
Ported aircraft loadouts
Removed nonfunctional transport plane
tweaked light helicopter lineup

**Review by killerswin2 — APPROVED** [review]
Me want snow gear. Approved. Assets might look like from OFP, but it should be fine.

**Pull request merged**